js/computation_graph.js

```diff
@@ -227,8 +226,8 @@ class InputNode extends ValueCheckComputeNode {
     constructor(name, input_field) {
         super(name);
         this.input_field = input_field;
-        this.input_field.addEventListener("input", () => calcSchedule(this, 500));
-        this.input_field.addEventListener("change", () => calcSchedule(this, 5));
+        this.input_field.addEventListener("input", () => { if (graph_live_update) calcSchedule(this, 500) } );
+        this.input_field.addEventListener("change", () => { if (graph_live_update) calcSchedule(this, 5) } );
```

Comment:
move the condition check to the input node (that way calcSchedule will always schedule calculation; just the input nodes will sometimes choose not to schedule calculation)